pwndbg/lib/regs.py

```diff
@@ -307,10 +384,45 @@ def __iter__(self) -> Iterator[str]:
 
 x86flags = {
     "eflags": BitFlags(
-        [("CF", 0), ("PF", 2), ("AF", 4), ("ZF", 6), ("SF", 7), ("IF", 9), ("DF", 10), ("OF", 11)]
+        [
+            ("CF", 0),
+            ("PF", 2),
+            ("AF", 4),
+            ("ZF", 6),
+            ("SF", 7),
+            ("IF", 9),
+            ("DF", 10),
+            ("OF", 11),
+            ("AC", 18),
+        ]
     )
 }
 
+amd64_kernel = KernelRegisterSet(
+    segments=["cs", "ss", "ds", "es", "fs", "gs"],
+    controls={
+        # only displays the security related bits, otherwise it can be too clustered
+        "cr0": BitFlags([("PE", 0), ("WP", 16), ("PG", 31)]),
+        "cr3": AddressingRegister(False),
+        "cr4": BitFlags(
+            [
+                ("UMIP", 11),
+                ("FSGSBASE", 16),
+                ("SMEP", 20),
+                ("SMAP", 21),
+                ("PKE", 22),
+                ("CET", 23),
+                ("PKS", 24),
+            ]
```

Comment:
All lists here (and above and below) should be tuples since we won't modify the flags length at runtime etc.